### ISSUE TITLE
Raspbian Image in the Gallery

### ISSUE THREAD

**Issue by krummrey** [utmapp/UTM#4154]
I've tried to get Raspian up and running on UTM. I failed. I was wondering if you could list it in the gallery. The real ones are hard to get these days and I'd love to try a few things out.

**Comment by jamesmortensen** [comment]
I wasn't able to get Raspbian working.  In general, I've not had a lot of luck with anything armv7 or armhf related except for this Debian VM I was able to create from scratch.  It's Debian though, not Raspbian.  https://github.com/jamesmortensen/virtual-machine-releases/releases/tag/debian-11.3-armhf-ci-v1.0

Aside from that, if you're really wanting to get your hands on Raspbian, I'm wondering if you could just get an installer ISO and try installing from scratch.  That's what I did with Debian 11 armhf...  I have it running in both UTM as well as raw QEMU.  Hope it helps.

**Comment by tangentsoft** [comment]
I got this working with Trixie (Debian 13). I'm attaching the config.plist file in case it helps.

The use case is for an ARMv7 build-bot for creating containers, via `podman farm build`. The only 32-bit ARM hardware I still have around hasn't got the resources to make it worth keeping it powered up versus running it in emulation. I selected Trixie so that I could have an up-to-date version of Podman.

The trickiest bit was finding a graphics setting it would work with. The default GPU settings didn't work, but that's fine because the box is basically headless.

Putting this in the gallery would be helpful. All I did before I started doing custom things in it was:

1. Next-next-next through the installer.
2. On initial boot, enable enp0s1 in `/etc/network/interfaces` to come up by default, with DHCP.
3. Install sudo.

I'd share my `.utm` file except that step 1 includes adding my users, and I gave it real passwords. I could redo the work with generic "gallery" settings, if that would be helpful.

[config.plist.zip](https://github.com/user-attachments/files/18532173/config.plist.zip)

**Comment by thegoodduck** [comment]
yes please, i have an Ipad Air 2 that is not powerful enough to run anything on utm(except react os and arch) and raspberry pi os should run pretty well so if you could recompile it it would be cool 😄

**Comment by kivikakk** [comment]
@tangentsoft thank you for this little rundown! For reference for others, a step-by-step:

* Grab an `armhf` ISO from the [Debian releases page](https://www.debian.org/releases/stable/debian-installer/). I used a netinst CD image.
* Create a new VM with these settings (sorry if I get the English names wrong):
  * System -> Architecture: ARM (aarch32)
  * System -> System: QEMU 10.0 ARM Virtual Machine (alias of virt-10.0) (virt)
  * Display -> Emulated graphics card: ram framebuffer standalone device (ramfb)
  * Network -> Network mode: Bridged (advanced)
* Leave the rest as their defaults; choose however much RAM and CPU cores as you like.
* Create a suitably sized disk to use as your hard drive. It should be a VirtIO disk image.
* Attach the ISO from above as another VirtIO disk image; set it to read-only.
* It should boot from the ISO; the installer should suggest installing only to the target.

It should Just Work™.
